### DISCUSSION TITLE
自动登录失败

### DISCUSSION THREAD

**Discussion by mushengtianqi** [lanyeeee/jmcomic-downloader#38] [Q&A]
0: Request error: error sending request for url (https://www.cdnxxx-proxy.org/login)
1: error sending request for url (https://www.cdnxxx-proxy.org/login)
2: operation timed out

请问这个是什么原因，是否有解决的办法，搜索漫画和下载也有类似情况

**Comment by lanyeeee** [comment]
这是网络原因，请求超时
解决办法：使用代理。如果已经使用代理了请尝试切换线路

这部分错误提示确实没写好，之后的版本会给出更友好的错误提示
我手动开了个issue #39
